### PR TITLE
v0.20.0 - Updating JavaScript exports and templates.

### PR DESCRIPTION
**Pull request by DamianMullins**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,11 +11,7 @@ sudo: false
 notifications:
   email: false
 
-before_script:
-  - yarn run lint
```
Comment (DamianMullins) [inline .travis.yml]: This script already runs as part of the `yarn prepare` script.

File: .travis.yml [hunk 2]
```diff
@@ -11,11 +11,7 @@ sudo: false
...
-
 script:
-  - yarn test:cover
```
Comment (DamianMullins) [inline .travis.yml]: Have added this to the `prepare` script.

File: package.json [hunk 3]
```diff
@@ -53,12 +53,11 @@
     "exports": false
   },
   "scripts": {
-    "prepare": "concurrently -n \"lint,compile,test\" -c \"blue,yellow,green\" \"yarn lint\" \"yarn compile\" \"yarn test\" --kill-others-on-fail",
+    "prepare": "concurrently -n \"lint,compile,test\" -c \"blue,yellow,green\" \"yarn lint\" \"yarn compile\" \"yarn test:cover\" --kill-others-on-fail",
     "lint": "yarn run lint:css && yarn run lint:js",
     "lint:css": "gulp scss:lint --prod",
     "lint:js": "gulp scripts:lint --prod",
     "compile": "babel -d dist src",
-    "test": "gulp scripts:test",
```
Comment (xander-marjoram) [inline package.json]: No more `yarn test`?

Comment (DamianMullins) [reply package.json]: You can still run it with gulp scripts:test, the npm script is on;y really there as a helper for the prepare script above.

Comment (DamianMullins) [reply package.json]: I can add it back in for consistency?

**Comment by DamianMullins** [comment]
I'm going to release a tagged beta of these changes as they are holding up another piece of work.

File: package.json (hunk 3 above)
Comment (ashleynolan) [reply package.json]: Is this removed because yarn test:cover does the same thing, or do we not want to run the tests on `prepare` now?

Comment (DamianMullins) [reply package.json]: Yeah `yarn test:cover` does the same thing, and also I imagine people are using the gulp scripts directly, right? The `test` script is kind of redundant here.

Comment (ashleynolan) [reply package.json]: Yeh, I just use the gulp scripts personally unless the yarn operation runs multiple things at once

**Pull request merged**